.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,62 @@
+---
+name: 🐛 Bug report
+description: Create a report to help us improve
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Before you continue filling out this report, please take a moment to check that your bug has not been [already reported on GitHub][issue search] 🙌
+
+        Remember to redact any sensitive information such as authentication credentials and/or license keys!
+
+        [issue search]: ../search?q=is%3Aissue&type=issues
+
```

Over the past couple days I have gone ahead and included the link to the community forum in the bug report too https://github.com/nginx/template-repository/commit/9518c542bdb36da9bd747d30ece1d819df293260
```suggestion

        **Note:** If you are seeking community support or have a question, please consider starting a new thread via [GitHub discussions][discussions] or the [NGINX Community forum][forum].

        [issue search]: ../search?q=is%3Aissue&type=issues
        [discussions]: ../discussions
        [forum]: https://community.nginx.org

```